src/Microsoft.ML.Core/Data/ColumnType.cs

```diff
@@ -789,6 +793,19 @@ public override bool Equals(ColumnType other)
             return true;
         }
 
+        public override bool Equals(Object other)
```

Comment:
>Equals [](start = 29, length = 6)

is it possible to have a test for it?

Reply:
yep, will do. Probably the biggest piece of code in this PR :)

---
In reply to: [206716330](https://github.com/dotnet/machinelearning/pull/620#discussion_r206716330) [](ancestors = 206716330)